test/css/component_specific_selectors_test.rb

```diff
@@ -45,6 +45,21 @@ class ComponentSpecificSelectorsTest < Minitest::Test
       ".Button--iconOnly.Button--small",
       ".Button--iconOnly.Button--large"
     ],
+    Primer::Beta::Avatar => [
+      ".avatar-link",
+      ".avatar-group-item",
+      ".avatar-1",
+      ".avatar-2",
+      ".avatar-3",
+      ".avatar-4",
+      ".avatar-5",
+      ".avatar-6",
+      ".avatar-7",
+      ".avatar-8"
```

Comment:
These sizes are not use in PVC and instead the `width/height` is inlined directly. I guess we could:

1. Use these classes for sizing in PVC
2. Move the styles to dotcom

Reply:
Hmm, looking at the react component, [they accept size in pixels](https://primer.style/react/Avatar), so I think we should move these to dotcom.

Reply:
Yeah. I guess we should also move `.avatar-link` and `.avatar-group-item` to dotcom. Or even replace them with utility classes since they are sparingly used.

Anyways, I think I'll leave that for another "cleanup unused CSS" round.